en/components/general/wpf_to_angular_guide/structural_directives.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Structural Directives in Angular
+_description: WPF to Angular guide topics help you switch from WPF to Angular.
+_keywords: WPF, Windows Presentation Foundation, Angular, Ignite UI for Angular
+---
+
+## Structural Directives in Angular
+
+As WPF developers, anytime we want to add or remove an element from the visual tree we have to jump into some code-behind and write some C# or we can use a combination of binding and a visibility converter, which again requires some custom logic and static resources. That is somewhat OK and we have always done it in WPF, but Angular makes it so much easier.
+Manipulating your view in an Angular application /adding and removing elements/, is achieved with the so-called "structural directives". In this topic, we are going to demonstrate the following three directives - `ngIf`, `ngSwitch` and `ngFor`. As one can tell from their names, each of these can be compared to a C# structure. The `ngIf` is the same thing as an "if-else" C# code block, the `ngSwitch` is the same thing as the C# switch-case statement and, lastly, the `ngFor` is the exact same thing as a C# "for-loop".
+
+### `ngIf` Directive
+Now, let’s explore each of these directives, starting with the `ngIf`. This directive allows us to show or hide elements based on a boolean condition. We will start by creating a "div" element with an "h2" tag containing a name.
+```html
+<div>
+    <h2>John</h2>
+</div>
+```
+If we save this our browser will render the name John. However, let’s say we have some type of boolean expression that we want to base the condition of the visibility of this "h2" tag on. For example, we are going to add a property called "isFirstName" and set it to false. In order to tell our div to be rendered when isFirstName equals true, we should use the following syntax *ngIf = "isFirstName".
+```typescript
+  public isFirstName = false;
```

Comment:
We can remove the indentation in the beginning.

Reply:
Addressed.